Packs/Devo/Integrations/Devo_v2/Devo_v2.yml

```diff
@@ -96,8 +91,7 @@ script:
           description: End datetime for specified query. If provided must be in same format as "from" argument. This argument is ignored in a date range.
         - name: items_per_page
           description: Enter the per page value you want to set.
-          required: false
-          defaultValue: "1000"
+          required: true
```

Comment:
The problem is that, depending on the data each row/record returns, the feasibility of returning the results or the docker container aborting the query is quite variable. In any way, let's lower the default value to 50 to avoid breaking backward compatibility.